src/convert.jl

```diff
@@ -102,8 +102,11 @@ function trixi2img(filename::AbstractString...;
       verbose && println("| Extracting 2D slice...")
       # convert 3d unstructured data to 2d slice
       @timeit "extract 2D slice" (unstructured_data, coordinates, levels,
-          center_level_0) = unstructured_2d_to_3d(unstructured_data, coordinates,
-          levels, length_level_0, center_level_0, slice_axis, slice_axis_intercept)
+          center_level_0, plot_labels) = unstructured_2d_to_3d(
+          unstructured_data, coordinates, levels, length_level_0, 
+          center_level_0, slice_axis, slice_axis_intercept)
+    elseif ndims == 2
+      plot_labels = ["x", "y"]
```

Comment:
You can use `Tuple`s for this, I think, i.e., `("x", "y")`. I think using tuples is generally preferred over arrays, since they do not allocate extra memory.